osu.Game/Screens/Edit/Compose/Components/BlueprintContainer.cs

```diff
@@ -505,7 +505,7 @@ private bool moveCurrentSelection(DragEvent e)
             Vector2 movePosition = movementBlueprintOriginalPositions.First() + distanceTravelled;
 
             // Retrieve a snapped position.
-            var result = snapProvider?.FindSnappedPositionAndTime(movePosition);
+            var result = snapProvider?.FindSnappedPositionAndTime(movePosition, ~SnapType.NearbyObjects);
```

Comment:
This, despite looking like it should never work, actually works because `~0 = -1` which returns true for `HasFlagFast(Grid)`.
```suggestion
            var result = snapProvider?.FindSnappedPositionAndTime(movePosition, SnapType.All & ~SnapType.NearbyObjects);
```
(or could just use `SnapType.Grids` and call it a day)